src/ecalc/cli/tests/snapshots/test_app/test_csv_resampled/results_resampled.csv

```diff
@@ -1,13 +1,13 @@
.hydrocarbon_export_rate[Sm3/d],model.co2.intensity_sm3[kg/Sm3],model.co2.intensity_boe[kg/BOE],model.co2.intensity_yearly_sm3[kg/Sm3],model.co2.intensity_yearly_boe[kg/BOE],Installation A.is_valid[N/A],Installation A.energy_usage[Sm3/d],Installation A.energy_usage_cumulative[Sm3],Installation A.power[MW],Installation A.power_cumulative[GWh],Installation A.co2.rate[t/d],Installation A.co2.cumulative[t],Installation A.co2.tax[NOK/d],Installation A.co2.tax_cumulative[NOK],Installation A.co2.quota[NOK/d],Installation A.co2.quota_cumulative[NOK],Installation A.hydrocarbon_export_rate[Sm3/d],Installation A.co2.intensity_sm3[kg/Sm3],Installation A.co2.intensity_boe[kg/BOE],Installation A.co2.intensity_yearly_sm3[kg/Sm3],Installation A.co2.intensity_yearly_boe[kg/BOE],Generator set A.is_valid[N/A],Generator set A.energy_usage[Sm3/d],Generator set A.energy_usage_cumulative[Sm3],Generator set A.power[MW],Generator set A.power_cumulative[GWh],Generator set A.co2.rate[t/d],Generator set A.co2.cumulative[t],Generator set A.co2.tax[NOK/d],Generator set A.co2.tax_cumulative[NOK],Generator set A.co2.quota[NOK/d],Generator set A.co2.quota_cumulative[NOK],Generator set A.power_capacity_margin[MW],Gas export compressor.is_valid[N/A],Gas export compressor.energy_usage[Sm3/d],Gas export compressor.energy_usage_cumulative[Sm3],Gas export compressor.co2.rate[t/d],Gas export compressor.co2.cumulative[t],Gas export compressor.co2.tax[NOK/d],Gas export compressor.co2.tax_cumulative[NOK],Gas export compressor.co2.quota[NOK/d],Gas export compressor.co2.quota_cumulative[NOK],Gas export compressor.recirculation_loss[MW],Gas export compressor.rate_exceeds_maximum[N/A],Gas export compressor.outlet_pressure_before_choking[bara],Gas injection compressor.is_valid[N/A],Gas injection compressor.energy_usage[MW],Gas injection compressor.energy_usage_cumulative[MWd],Gas injection compressor.power[MW],Gas injection compressor.power_cumulative[GWh],Gas injection compressor.recirculation_loss[MW],Gas injection compressor.rate_exceeds_maximum[N/A],Gas injection compressor.outlet_pressure_before_choking[bara],Produced water reinjection pump.is_valid[N/A],Produced water reinjection pump.energy_usage[MW],Produced water reinjection pump.energy_usage_cumulative[MWd],Produced water reinjection pump.power[MW],Produced water reinjection pump.power_cumulative[GWh],Base production load.is_valid[N/A],Base production load.energy_usage[MW],Base production load.energy_usage_cumulative[MWd],Base production load.power[MW],Base production load.power_cumulative[GWh],Flare.is_valid[N/A],Flare.energy_usage[Sm3/d],Flare.energy_usage_cumulative[Sm3],Flare.co2.rate[t/d],Flare.co2.cumulative[t],Flare.co2.tax[NOK/d],Flare.co2.tax_cumulative[NOK],Flare.co2.quota[NOK/d],Flare.co2.quota_cumulative[NOK],Sea water injection pump.is_valid[N/A],Sea water injection pump.energy_usage[MW],Sea water injection pump.energy_usage_cumulative[MWd],Sea water injection pump.power[MW],Sea water injection pump.power_cumulative[GWh]
```

Comment:
Several changes that should not be here (CO2): Value differences in 2024 and 2031, and nan-values in 2025. Why?